crates/loro-internal/src/container/richtext/richtext_state.rs

```diff
@@ -1215,7 +1239,7 @@ impl RichtextState {
         }
 
         let (c, entity_index) = match pos_type {
-            PosType::Bytes => todo!(),
+            PosType::Bytes => self.find_best_insert_pos::<UnicodeQueryT>(pos),
```

Comment:
You may need to create a `ByteQueryT`

Reply:
I have updated the code so that the pos here has been converted to a unicode pos, because Byte's pos cannot also cut unicode characters, so I think `ByteQueryT` is not needed at the moment.

Reply:
In that case, you can call `get_entity_index_for_text_insert(pos, PosType::Unicode)` on the call site and leave this part as `PosType::Bytes => unreachable!()`. It's less confusing and less error-prone